runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPlugin.java

```diff
@@ -215,6 +232,53 @@ public void onConfigChanged(ConfigChanged event)
 		clientThread.invokeLater(() -> checkRaidPresence(true));
 	}
 
+	@Subscribe
+	public void onWidgetLoaded(WidgetLoaded event)
+	{
+		if (!config.showLootValue() ||
+			event.getGroupId() != WidgetID.CHAMBERS_OF_XERIC_REWARD_GROUP_ID ||
+			chestOpened)
+		{
+			return;
+		}
+		chestOpened = true;
+		ItemContainer rewardItemContainer = client.getItemContainer(InventoryID.CHAMBERS_OF_XERIC_CHEST);
+		if (rewardItemContainer == null)
+		{
+			return;
+		}
+
+		final Collection<Item> rewardItems = Arrays.stream(rewardItemContainer.getItems())
+			.filter(item -> item.getId() > 0)
+			.map(item -> new Item(item.getId(), item.getQuantity()))
+			.collect(Collectors.toList());
+
+		int rewardValue = 0;
```

Comment:
This should maybe be a `long`.

Reply:
QuantityFormatter.quantitotyToRsDecimalStack requires an int, that is why I left it as int